### PR TITLE
 [Getting Started Videos] Add Ons + Videos 

### PR DESCRIPTION
**Pull request by KatyCain526**
Box, Google, Drive, Zotero, Mendeley, Wiki

### CODE REVIEW

File: website/templates/public/pages/help/addons.mako [hunk 1]
```diff
@@ -67,3 +67,49 @@
 <p>Next, authenticate with Dataverse by entering your Dataverse username and password and clicking "Submit". You can then choose the Dataverse study you would like to add to your OSF project. Click "Submit" to save your settings.</p>
 <p>Contributors to your project will have access to both released and draft versions of your study, but only the most recent release will be made public alongside your OSF project.</p>
 
+<span id="box" class="anchor"></span>
+<p class="gs-header">Box</p>
+<p><strong><a href="http://box.com/">Box</a> is an online file storage program.</strong>AAfter linking your OSF project to Box, you can add files to your Box folder and those files can be accessed via the OSF. Likewise, files added to the Box folder in your OSF account will update your Box account with that file.</p>
+<p>To link a Box folder to a project/component, first visit the project you want to add a Box folder to. Then go to "Settings" in the grey navigation bar. Check “Box" under "Select Add-ons" to enable the add-on. Read, click "OK" on the pop-up, then submit.</p>
```
Comment (sloria) [inline website/templates/public/pages/help/addons.mako]: Remove smart quotes.

Comment (sloria) [inline website/templates/public/pages/help/addons.mako]: "Box"

**Comment by sloria** [comment]
Still getting an encoding error from addons.mako

**Comment by KatyCain526** [comment]
@sloria I think I got it.

**Comment by sloria** [comment]
:octopus:

File: website/templates/public/pages/help/addons.mako [hunk 2]
```diff
@@ -67,3 +67,43 @@
...
+<p>Now you will be able to find your Google Drive folder in your "Files" section. </p>
+
+
+<h3> Citation Managers </h3>
+<div class="gs-video embed-responsive embed-responsive-16by9">
+    <div class="embed-responsive-item youtube-loader" id="2RVJS7nYX8k"></div>
+</div>
+
+
+<span id="zotero" class="anchor"></span>
```
Comment (sloria) [inline website/templates/public/pages/help/addons.mako]: The ordering with the side navbar is different; Mendeley goes before Zotero. I will fix this when I merge.

**Pull request merged**